Events/Ch9.event

```diff
@@ -33,7 +33,7 @@ UNIT 0x92 Mogall Carlyle Level(12, Enemy, 1) [20,13] 0x00 0x00 0x0 0x00 [EvilEye
 UNIT 0x92 Mogall Carlyle Level(12, Enemy, 1) [18,11] 0x00 0x00 0x0 0x00 [EvilEye] AttackInRangeAI
 UNIT 0x92 Gargoyle Carlyle Level(12, Enemy, 1) [4,14] 0x2 0x00 0x0 0x00 [Axereaver] AttackInRangeAI
 UNIT 0x92 Gargoyle Carlyle Level(12, Enemy, 1) [6,12] 0x00 0x00 0x0 0x00 [Javelin] AttackInRangeAI
-UNIT 0x92 Tarvos Carlyle Level(12, Enemy, 1) [5,16] 0x2 0x00 0x0 0x00 [DevilAxe] AttackInRangeAI
+UNIT 0x92 Tarvos Carlyle Level(12, Enemy, 1) [5,16] 0x2 0x00 0x0 0x00 [DevilAxe] AttackInRangeAI //Colorz said Devil Axe is nonfunctional/not to be used (?)
```

Comment:
Is currently bugged with the skill system. VBA1 had them only by removing the devil effect from it. If you're doing that anyway, probably just give them a silver axe.

Reply:
(i'm not editing somebody else's chapter though)